webapp-django/crashstats/crashstats/static/crashstats/js/socorro/correlation.js

```diff
@@ -136,10 +136,36 @@ window.correlations = (function () {
                 return 1;
             }
 
+            // If one of the two elements has a prior that alters a rule's distribution significantly, sort by the
+            // percentage of the rule given the prior.
+            if (!a.prior) {
+                var count_group_a = a.count_group;
+                var total_group_a = total_group;
+                var count_reference_a = a.count_reference;
+                var total_reference_a = total_reference;
+            } else {
+                var count_group_a = a.prior.count_group;
+                var total_group_a = a.prior.total_group;
+                var count_reference_a = a.prior.count_reference;
+                var total_reference_a = a.prior.total_reference;
+            }
+
+            if (!b.prior) {
+                var count_group_b = b.count_group;
+                var total_group_b = total_group;
+                var count_reference_b = b.count_reference;
+                var total_reference_b = total_reference;
+            } else {
+                var count_group_b = b.prior.count_group;
+                var total_group_b = b.prior.total_group;
+                var count_reference_b = b.prior.count_reference;
+                var total_reference_b = b.prior.total_reference;
+            }
```

Comment:
There's a lot of repetition here and I don't know what the latest linting rules think about declaring the same named variable in the same scope. 
I think all of these could be refactored to something like `var count_group_a = a.prior && a.prior.count_group || a.count_group`
Or you can rewrite `a` to so something like this:
```javascript
if (a.prior) {
    a = a.prior;
}
var count_group_a = a.count_group;
...
```